src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
you need `min-width` 😉

<img width="682" alt="image" src="https://user-images.githubusercontent.com/55802630/173025578-c590b2cc-943d-41f5-a9e7-0570c733980c.png">

<img width="422" alt="image" src="https://user-images.githubusercontent.com/55802630/173025610-b98d0d70-61b9-41f4-99c6-8339db52cf42.png">